c_common/models/system_models/src/extra_monitor_support.c

```diff
@@ -148,14 +137,17 @@ typedef enum data_out_sdp_commands {
 #define TDMA_WAIT_PERIOD   0
 
 //! The initial timeout of the router
-#define ROUTER_INITIAL_TIMEOUT 0x004f0000
+#define ROUTER_INITIAL_TIMEOUT 0x4f
```

Comment:
is that actually correct? given the number has changed. just in case its working, but working for the wrong reasons

Reply:
It's the initial value, not necessarily the best option. (The change of the value here is because it has been disaggregated from the containing structure. And I can never decode those weird 8-bit float things in my head anyway.)

Reply:
fiar enough then